src/biokbase/narrative/jobs/jobcomm.py

```diff
@@ -28,58 +30,54 @@ class JobRequest(object):
     """
     A small wrapper for job comm channel request data.
     This generally comes in the form of a packet from the kernel Comm object.
-    It's expected to be a dict of the format:
+    It is expected to be a dict of the format:
     {
         content: {
             'comm_id': <some string>,
             'data': {
-                'request_type': effectively, the function requested.
+                'request_type': job function requested
                 'job_id': optional
                 'job_id_list': optional
+                'batch_id': optional
             }
         }
     }
 
-    This little wrapper fills 2 roles:
+    This little wrapper fills some roles:
     1. It validates that the packet is correct and has some expected values.
-    2. If there's a job_id field, it makes sure that it's real (by asking the
-       JobManager - avoids a bunch of duplicate validation code)
-
-    Each JobRequest has at most one of a job_id or job_id_list. If the job comm
-    channel request data is received with a job_id_list, it may be split up
-    into multiple JobRequests. Likewise, if the job comm channel request data
-    is received with a job_id, a JobRequest may be created with a job_id_list
-    containing that job_id
+    2. It provides any job ID information with guardrails
```

Comment:
?? guardrails?

Reply:
Just trying to delete obsolete documentation and smooth it over. The guardrails are in the code